src/ZODB/multicommitadapter.py

```diff
@@ -0,0 +1,65 @@
+"""Adapt non-IMultiCommitStorage storages to IMultiCommitStorage
+"""
+
+import zope.interface
+
+from .ConflictResolution import ResolvedSerial
+
+class MultiCommitAdapter:
+
+    def __init__(self, storage):
+        self._storage = storage
+        ifaces = zope.interface.providedBy(storage)
+        zope.interface.alsoProvides(self, ifaces)
+        self._resolved = set()
```

Comment:
It is, but some reviewers' linters complain if variables aren't initialized, and I prefered here to initialize with a valid value.